src/pages/api/stock/[ticker].ts

```diff
@@ -21,6 +21,7 @@ const endpoint = async (
 
   const prisma = new PrismaClient();
 
+  console.log(ticker.toString());
```

Comment:
maybe add a todo or comment here so we don't forget to remove the console log before submission, otherwise v minor can ignore